code_samples/additional-info/rest-api/index.js

```diff
@@ -1,9 +1,10 @@
-const { Client, RichEmbed } = require('discord.js');
-const { get } = require('snekfetch');
+const snekfetch = require('snekfetch');
+const Discord = require('discord.js');
 const { prefix, token } = require('./config.json');
```

Comment:
No config file so it should be removed, as well as have a hard-coded prefix variable.